packages/backend-modules/documents/graphql/schema-types.js

```diff
@@ -103,6 +108,9 @@ type Meta {
   indicateChart: Boolean
   indicateGallery: Boolean
   indicateVideo: Boolean
+  
+  paynotes: [JSON]
+  tryToBuyRatio: Float
```

Comment:
The values we actually return are `null`, `1` and `0` right? Maybe it would be better for have a `paynoteMode` as enum with e.g. `auto`, `try`, `buy` and `none`?

Reply:
Good tip. I went for a `paynoteCta` construct, which does clarify the code somewhat.